esphome/components/toshiba/toshiba.cpp

```diff
@@ -490,7 +1038,155 @@ bool ToshibaClimate::is_valid_rac_pt1411hwru_message_(const uint8_t *message) {
   return false;
 }
 
+bool ToshibaClimate::process_ras_2819t_command_(const remote_base::ToshibaAcData &toshiba_data) {
+  // Check for power-off command (single packet)
+  if (toshiba_data.rc_code_2 == 0 && toshiba_data.rc_code_1 == RAS_2819T_POWER_OFF_COMMAND) {
+    ESP_LOGI(TAG, "RAS-2819T power OFF command detected!");
```

Comment:
This is probably ok to leave as "debug" level, but let's shorten it per [our guidelines](https://developers.esphome.io/architecture/logging/#examples):
```suggestion
    ESP_LOGD(TAG, "Received RAS-2819T power OFF");
```